smarts/env/gymnasium/wrappers/metrics.py

```diff
@@ -100,12 +82,20 @@ def __init__(self, env: gym.Env):
 
     def step(self, action: Dict[str, Any]):
         """Steps the environment by one step."""
-        obs, rewards, dones, info = super().step(action)
+        result = super().step(action)
+
+        terminated, truncated = False, False
+        obs, _, terminated, truncated, info = result
 
         # Only count steps in which an ego agent is present.
         if len(obs) == 0:
-            return obs, rewards, dones, info
+            return result
 
+        done = False
+        if terminated or truncated:
+            done = True
```

Comment:
```suggestion
        done = terminated or truncated
```

Reply:
This has been set to configure the dictionaries version of done.